### PR TITLE
Fix the FSx Readme and update auto-script

### PR DESCRIPTION
**Pull request by mbaijal**
**Which issue is resolved by this Pull Request:**
Update the FSx Readme and automated script 

**Description of your changes:**
- Updated the auto-script to also automate the PV creation since there are some manual steps
- Cleaned up README such that it works smoothly with the automated script. 

Link to test: https://gist.github.com/mbaijal/bc14c921270141c557d0ee348753012e
TODO:
 - https://github.com/awslabs/kubeflow-manifests/issues/192

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/deployment/add-ons/storage/fsx-for-lustre/README.md [hunk 1]
```diff
@@ -5,33 +5,70 @@ This guide describes how to use Amazon FSx as Persistent storage on top of an ex
...
 
-2. At this point, you have likely cloned this repo and checked out the right branch. Navigate to the current directory - 
+**Important :**
+You must make sure you have an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for your cluster and that it was added from `eksctl` >= `0.56` or if you already have an OIDC provider in place, then you must make sure you have the tag `alpha.eksctl.io/cluster-name` with the cluster name as its value. If you don't have the tag, you can add it via the AWS Console by navigating to IAM->Identity providers->Your OIDC->Tags.
+
+2. At this point, you have likely cloned this repo and checked out the right branch. Let's save this path to help us naviagte to different paths in the rest of this doc -
```
Comment (ryansteakley) [inline docs/deployment/add-ons/storage/fsx-for-lustre/README.md:11]: nit: naviagte -> navigate

File: docs/deployment/add-ons/storage/fsx-for-lustre/README.md [hunk 2]
```diff
@@ -5,33 +5,70 @@ This guide describes how to use Amazon FSx as Persistent storage on top of an ex
...
+
+Note: If you want the script to create a new security group for FSx, specify a name for `SECURITY_GROUP_TO_CREATE`. On the other hand, if you want to use an existing Security group, you can specify that name too.
+```
+export SECURITY_GROUP_TO_CREATE=$CLAIM_NAME
+
+python utils/auto-fsx-setup.py --region $CLUSTER_REGION --cluster $CLUSTER_NAME --fsx_file_system_name $CLAIM_NAME --fsx_security_group_name $SECURITY_GROUP_TO_CREATE
+```
+
+4. The script above takes care of creating the `PersistentVolume (PV)` which is a cluster scoped resource. In order to create the `PersistentVolumeClaim (PVC)` you can either use the yaml file provided in this directory or use the Kubeflow UI directly. 
+The PVC needs to be in the namespace you will be accessing it from. Replace the `kubeflow-user-example-com` namespace specified the below with the namespace for your kubeflow user and edit the `fsx-for-lustre/static-provisioning/pvc.yaml` file accordingly.
```
Comment (ryansteakley) [inline docs/deployment/add-ons/storage/fsx-for-lustre/README.md:56]: Nit: 
> Replace the `kubeflow-user-example-com` namespace specified the below with the namespace for your kubeflow user
extra the in this sentence hard to read

File: docs/deployment/add-ons/storage/fsx-for-lustre/README.md [hunk 3]
```diff
@@ -134,31 +172,32 @@ NAME        STATUS   VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
...
+### 3.2 Note about Permissions
+This step may not be necessary but you might need to specify some additional directory permissions on your worker node before you can use these as mount points. By default, new Amazon EFS file systems are owned by root:root, and only the root user (UID 0) has read-write-execute permissions. If your containers are not running as root, you must change the Amazon EFS file system permissions to allow other users to modify the file system. The set-permission-job.yaml is an example of how you could set these permissions to be able to use the efs as your workspace in your kubeflow notebook. Modify it accordingly if you run into similar permission issues with any other job pod.
```
Comment (ryansteakley) [inline docs/deployment/add-ons/storage/fsx-for-lustre/README.md:183]: Some references to efs here should be FSx?

**Review by ryansteakley — COMMENTED** [review]
Minor comments thanks for taking the time to work on this

**Review by ryansteakley — APPROVED** [review]

**Review by goswamig — COMMENTED** [review]
can u paste the test result output?

**Review by goswamig — APPROVED** [review]

**Pull request merged**